web/controllers/certificateController/index.ts

```diff
@@ -151,4 +201,30 @@ function createPDFBinary(student: Student, pupil: Pupil, startDate: Date, params
             }
         });
     });
-}
\ No newline at end of file
+}
+
+async function viewParticipationCertificate(certificateId) {
+    const entityManager = getManager();
+    let certificate = null;
+    let html = readFileSync("./assets/verifiedCertificatePage.html", "utf8");
+    try {
+        certificate = await entityManager.findOne(ParticipationCertificate, { uuid: certificateId },  { relations: ["student", "pupil"] });
```

Comment:
Probably we can add some case insensitivity here when querying the database for a certificate ID. This is only additional and not necessary ("nice to have"), but case insensitive names could probably simplify typing the url from a printed certificate paper.

Reply:
done [https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b](https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b)